.github/workflows/mojas-archive-repos.yml

```diff
@@ -0,0 +1,28 @@
+name: MoJ Analytical Services - Archive repo if over a certain date
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 12 * * 1-5"
+
+jobs:
+  archive-repos:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+      - run: pip install -r python/requirements.txt
+      - run: python -m python.scripts.mojas.archive_repositories
```

Comment:
should this be` python.mojas.archive_repositories` ?

Reply:
Yes! Good spot! probably saved a couple of hours of dubbing that! 😂